DATA/production/qc-async/mid.json

```diff
@@ -3,23 +3,18 @@
     "config": {
       "database": {
         "implementation": "CCDB",
-        "host": "ali-qcdb.cern.ch:8083",
-        "username": "not_applicable",
-        "password": "not_applicable",
-        "name": "not_applicable"
+            "host": "ccdb-test.cern.ch:8080"
```

Comment:
Why use the test instance here (and also below in line 17)?
Shouldn't it be the production instance `alice-ccdb`?

Reply:
These were recommendations advised by me and Barth during QC module reviews we did recently. We suggested that test values are used in there, so that when testing, the results are not stored in the production database. When running on grid, these values are anyway overwritten to prod.

Reply:
also, prod url for QCDB is not `alice-ccdb`, we store QC objects in a separate CCDB instance.